frontend/public/components/edit-yaml.jsx

```diff
@@ -112,6 +113,20 @@ export const EditYAML_ = connect(stateToProps)(
         }
 
         createResources(objs, isDryRun) {
+          // If a namespace is being created, wait for each request synchronously.
```

Comment:
I would just always run the requests synchronously. Create requests should be fast, and the list should never be extremely large.